x/cert/client/rest/query.go

```diff
@@ -180,10 +180,9 @@ func certificatesHandler(cliCtx client.Context) http.HandlerFunc {
 			}
 		}
 
-		contentType := r.URL.Query().Get("requestcontenttype")
-		content := r.URL.Query().Get("requestcontent")
+		CertType := r.URL.Query().Get("certificatetype")
```

Comment:
Uppercase variable name by mistake I suppose?

Reply:
Oops yes, it was fixed.